src/baseline_bots/utils.py

```diff
@@ -396,20 +436,80 @@ def dipnetify_suborder(suborder: str) -> str:
     elif daide_style_order_groups[1] == "CVY":
         # Convoy order
         dipnet_order.append("C")
-        dipnet_order.append(dipnetify_suborder(daide_style_order_groups[2]))
+        dipnet_order.append(dipnetify_suborder(daide_style_order_groups[2])[0])
         dipnet_order.append("-")
         dipnet_order.append(daide_style_order_groups[4])
     elif daide_style_order_groups[1] == "MTO":
         # Move orders
         dipnet_order.append("-")
         dipnet_order.append(daide_style_order_groups[2])
         if len(daide_style_order_groups) > 3:
-            raise f"error from utils.daide_to_dipnet_parsing: order {daide_style_order_groups} is UNEXPECTED. Update code to handle this case!!!"
```

Comment:
Why is this being removed? The print then exception pattern should be avoided

Reply:
Because the f strings / formatted strings / string concatenation are not working while raise Exceptions. So, we won't be able to see the variable value.

Reply:
Or we would have to completely shift to the logger pattern. But might be of lesser priority compared to the other tasks for the eval.

Reply:
They should work fine. Have you tried

```python            
raise Exception(f"error from utils.daide_to_dipnet_parsing: order {daide_style_order_groups} is UNEXPECTED. Update code to handle this case!!!")
```

Reply:
Ah this works. Thanks. Replaced.